doc/cephfs/disaster-recovery-experts.rst

```diff
@@ -241,8 +241,8 @@ recovery since the existing metadata pool would not be modified.
 
       ``<fs_name>`` here and below refers to the original, damaged file system.
 
-#. Next, create a recovery file system in which we will populate a new metadata
-   pool that is backed by the original data pool:
+#. Create a recovery file system. This recovery file system will be fitted with
```

Comment:
`fitted` might not be known to some of our audience; weakly suggest "deployed"

Reply:
That's a good point. I agonized over this, and "fitted" was the best I could come up with. I'm grateful for your fresh eyes.

Reply:
`fitted` to me is something done nasally to fire, ymmv.

Reply:
I have spent too long thinking about this. But here is what I think. I think "deployed" isn't quite right. When we create the file system, it doesn't have the data pool in it. We have to undertake another procedure in order to deploy the data pool in the newly created filesystem. So I'm going to write something like ``This recovery file system will be used to recover the data in the damaged pool. First, the filesystem will have a data pool deployed to it. Then you will attach a new metadata pool to the new data pool. Then you will set the new metadata pool to be backed by the old metadata pool.`` 

Let me know what you think.

Reply:
Sure, though I suggest "deployed for it"

Reply:
We'll go with that. You know your way around a data pool better than I do.